### ISSUE TITLE
Train

### ISSUE THREAD

**Issue by nowander** [etoilestar/GCNv2-project#2]
In data.py the line 283  train_data = DataLoader(MYdata(params['imagepath'], 'data_keypoint1.csv', mode='train'),1, shuffle=False, num_workers=1)
where is the data_keypoint1.csv.

If possible, could you upload '../img/file' and also write the train instructions? Thank you.
